cmd/clustertree/cluster-manager/app/options/options.go

```diff
@@ -70,6 +73,7 @@ func (o *Options) AddFlags(flags *pflag.FlagSet) {
 	flags.BoolVar(&o.MultiClusterService, "multi-cluster-service", false, "Turn on or off mcs support.")
 	flags.StringVar(&o.RootCoreDNSServiceNamespace, "root-coredns-service-namespace", CoreDNSServiceNamespace, "The namespace of the CoreDNS service in the root cluster, used to locate the CoreDNS service when MultiClusterService is disabled.")
 	flags.StringVar(&o.RootCoreDNSServiceName, "root-coredns-service-name", CoreDNSServiceName, "The name of the CoreDNS service in the root cluster, used to locate the CoreDNS service when MultiClusterService is disabled.")
+	flags.BoolVar(&o.StorageOnewayController, "storage-oneway-controller", false, "Turn on or off storage oneway controllers.")
```

Comment:
Storage-oneway-controller seems not consist with `oneway_pv_controller.go`.
We can name the switch `Oneway-Storage-controller` or name the controller `pv_oneway-controller` for the consistence

Reply:
Alright, I've modified `storage-oneway-controller` to `oneway-storage-controllers` because `PvOnewayController` was triggering a 'Name starts with the package name' warning.